app/views/api/v0/organizations/articles.json.jbuilder

```diff
@@ -0,0 +1,14 @@
+json.array! @articles do |article|
```

Comment:
can't we just render the `index.json.jbuilder` file from the articles API in the action of this? They are identical except for the if, which in this case it will always be true :-)

Reply:
Thanks for reviewing. I changed it to render the index jbuilder.